### DISCUSSION TITLE
Access Denied error

### DISCUSSION THREAD

**Discussion by gjowen2255** [carson-katri/dream-textures#59] [Q&A] [answered]
Hi, I'm getting this error at install

PermissionError: [WinError 5] Access is denied: 'C:\\Program Files\\Blender Foundation\\Blender 3.3\\3.3\\python\\Include'

**Comment by Umoy** [comment, marked as answer]
hi,

try to run blender as an admisitrator. This could help.

Greetz
Umoy

**Reply by gjowen2255** [reply]
That did the trick, thanks!
